specification/machinelearningservices/AzureAI.Assets/common.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+namespace Microsoft.MachineLearningServices;
+
+// Workspace definitions are to correctly nest resources under workspaces (not to redefine the definitions in swagger)
+@doc("Machine learning workspace.")
+@Autorest.useRef("../../../../resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/workspaceRP.json#/definitions/Workspaces")
+@resource("workspaces")
+model Workspace {
```

Comment:
This looks like a resource manager resource, is this a data plane or a control plane API?